src/database.js

```diff
@@ -6,56 +6,66 @@ const { DB_USER, DB_PASSWORD, DB_HOST, DB_PORT, DB_NAME } = process.env
 const sequelize = new Sequelize(`postgres://${DB_USER}:${DB_PASSWORD}@${DB_HOST}:${DB_PORT}/${DB_NAME}`, {
   logging: false,
   native: false
-})
+});
 
 // Imports models
-const EnterpriseModel = require('./models/Enterprise.js')
-const ReserveModel = require('./models/Reserve.js')
-const RoleModel = require('./models/Role.js')
-const TripModel = require('./models/Trip.js')
-const TripAdditionalChargeModel = require('./models/TripAdditionalCharge.js')
-const UserModel = require('./models/User.js')
+const EnterpriseModel = require('./models/Enterprise.js');
+const ReserveModel = require('./models/Reserve.js');
+const DriverModel = require('./models/Driver.js');
+const TripModel = require('./models/Trip.js');
+const StopModel = require('./models/Stop.js');
+const UserModel = require('./models/User.js');
+const FinanceDriverModel = require('./models/FinanceDriver.js');
+const FinanceEnterpriceModel = require('./models/FinanceEnterprice.js');
```

Comment:
Enterprise

Reply:
Could you elaborate more? I don't understand what you mean.